### PR TITLE
Converted exisiting translated components to using composeMixin

### PR DESCRIPTION
**Pull request by MitchelPaulin**

### CODE REVIEW

File: src/components/rubric-alignments-indicator.js [hunk 1]
```diff
@@ -13,8 +14,14 @@ import { LocalizeMixin } from '../mixins/localize-mixin.js';
 import { OutcomeEntity } from '../entities/outcome-entity.js';
 import { RubricCriterionActivityUsageEntity } from '../entities/rubric-criterion-activity-usage-entity.js';
 import { rubricStyles } from '../styles/rubric-styles.js';
+EntityWatcherMixin(LocalizeMixin(EntityMixin(LitElement)))
```
Comment (devpow112) [inline src/components/rubric-alignments-indicator.js:17]: I'm guessing this was a copy/paste error?

Comment (MitchelPaulin) [reply src/components/rubric-alignments-indicator.js]: Correct

**Comment by devpow112** [comment]
You can actually use the `composeMixins` from brightspace ui core now. Brandon was good enough to port it over.
https://github.com/BrightspaceUI/core/blob/main/helpers/composeMixins.js

File: src/components/rubric-alignments-indicator.js (hunk 1 above)
Comment (MitchelPaulin) [reply src/components/rubric-alignments-indicator.js]: Funnily enough the linter actually caught it because I was missing a semi colon lol

**Review by devpow112 — APPROVED** [review]

**Pull request merged**